pay-api/src/pay_api/utils/enums.py

```diff
@@ -89,6 +89,10 @@ class PaymentMethod(Enum):
     DRAWDOWN = 'DRAWDOWN'
     INTERNAL = 'INTERNAL'
     DIRECT_PAY = 'DIRECT_PAY'
+    EFT = 'EFT'
+    WIRE = 'WIRE'
+    ONLINE_BANKING = 'OB'
```

Comment:
is OB a right term , is online banking generally short termed as OB? or should we just keep ONLINE_BANKING itself ? i don't specifically mean here; but in database as well...

Reply:
fixed